packages/ui/src/composables/std/internal/useUserProvidedProp.ts

```diff
@@ -22,6 +22,8 @@ export const useIsUserProvidedProp = (propName: string) => {
     if (!vm?.vnode.props) { return false }
 
     // If vnode doesn't have this prop it mean default value is used
+    // Note: be cautious as the `propName` should have the exact case in the template
```

Comment:
UserProvidedProp is used outside of Vuestic internals. We need to to work with both cases.